.github/workflows/nodejs.yml

```diff
@@ -17,15 +17,76 @@ jobs:
-          #- 20.x
-          #- 22.x
+        inlude:
+          - node-version: 16.x
+            test-workspaces: >-
+              --workspace=packages/aa
+              --workspace=packages/allow-scripts
+              --workspace=packages/lavapack
+              --workspace=packages/preinstall-always-fail
+              --workspace=packages/tofu
+              --workspace=packages/yarn-plugin-allow-scripts
+          - node-version: 18.x
+            test-workspaces: >-
+              --workspace=packages/aa
+              --workspace=packages/allow-scripts
+              --workspace=packages/browserify
+              --workspace=packages/core
+              --workspace=packages/lavapack
+              --workspace=packages/laverna
+              --workspace=packages/node
+              --workspace=packages/perf
+              --workspace=packages/preinstall-always-fail
+              --workspace=packages/survey
+              --workspace=packages/tofu
+              --workspace=packages/webpack
+              --workspace=packages/yarn-plugin-allow-scripts
+
+          # TODO: Resolve https://github.com/LavaMoat/LavaMoat/issues/1211 and enable v20.x, v22.x
+          - node-version: 20.14.x
+            test-workspaces: >-
+              --workspace=packages/aa
+              --workspace=packages/browserify
+              --workspace=packages/core
+              --workspace=packages/node
+              --workspace=packages/tofu
+              --workspace=packages/webpack
+          # TODO: Resolve https://github.com/LavaMoat/LavaMoat/issues/1211 and enable v20.x, v22.x
+          - node-version: 22.1.x
+            test-workspaces: >-
+              --workspace=packages/aa
+              --workspace=packages/browserify
+              --workspace=packages/core
+              --workspace=packages/node
+              --workspace=packages/tofu
+              --workspace=packages/webpack
+
+          - node-version: 20.x
+            test-workspaces: >-
+              --workspace=packages/aa
+              --workspace=packages/allow-scripts
+              --workspace=packages/core
+              --workspace=packages/lavapack
+              --workspace=packages/laverna
+              --workspace=packages/perf
+              --workspace=packages/preinstall-always-fail
+              --workspace=packages/survey
+              --workspace=packages/tofu
+              --workspace=packages/yarn-plugin-allow-scripts
+          - node-version: 22.x
+            test-workspaces: >-
+              --workspace=packages/aa
+              --workspace=packages/allow-scripts
+              --workspace=packages/core
+              --workspace=packages/lavapack
+              --workspace=packages/laverna
+              --workspace=packages/perf
+              --workspace=packages/preinstall-always-fail
+              --workspace=packages/survey
+              --workspace=packages/tofu
+              --workspace=packages/yarn-plugin-allow-scripts
```

Comment:
(could) If there's a way to do it, having a variable called ALL_WORKSPACES here would be cool looking.